### ISSUE TITLE
Detect map coordinates for player bases

### ISSUE THREAD

**Issue by tylercamp** [tylercamp/palcalc#9]
Rather than just saying "a base" as the location for pals currently deployed to a base, it would be much better to show the map coordinates for that base.

The Palworld save file includes a list of all bases, their coordinates, and the pal container ID associated with them. The coordinates for a base are in world coordinates, not map coordinates, and the "map object" IDs associated with a base all seem to be set to the same position and are not useful for calculating the map coordinates of a base.

I think we'll need to figure out the transform from world coordinates to map coordinates and reflect them in Pal Calc.

It would be helpful to have some examples of bases with their world + map coordinates for reference.

**Comment by tylercamp** [comment]
I found these properties in `Pal/Content/Pal/DataTable/WorldMapUIData/DT_WorldMapUIData.uasset`:

```
        "landScapeRealPositionMin": {
          "X": -582888.0,
          "Y": -301000.0,
          "Z": 1.0
        },
        "landScapeRealPositionMax": {
          "X": 335112.0,
          "Y": 617000.0,
          "Z": 1.0
        },
```

**Comment by tylercamp** [comment]
Looks like the map coord calc for _any_ set of world coords is:

```js
const MapMinX = -582888.0
const MapMinY = -301000.0

const MapMaxX = 335112.0
const MapMaxY = 617000.0

const norm = (v, min, max) => (v - min) / (max - min)
const xnorm = x => norm(v, MapMinX, MapMapX)
const ynorm = y => norm(v, MapMinY, MapMaxY)

const mapX = 1000 * (ynorm(worldY) * 2 - 1)
const mapY = 1000 * (xnorm(worldX) * 2 - 1)
```

(X and Y world coords are swapped when presented as map coords)

Tried this manually with my own save and it seems correct so far

**Comment by tylercamp** [comment]
Released in v1.9.0
